components/supervisor/frontend/src/supervisor-service-client.ts

```diff
@@ -27,16 +20,26 @@ export class SupervisorServiceClient {
         if (kind == "supervisor") {
             wait = "";
         }
-        return fetch(window.location.protocol + '//' + window.location.host + '/_supervisor/v1/status/' + kind + wait, { credentials: 'include' }).then(response => {
+        try {
+            const response = await fetch(window.location.protocol + '//' + window.location.host + '/_supervisor/v1/status/' + kind + wait, { credentials: 'include' });
+            let result;
             if (response.ok) {
-                return;
+                result = await response.json();
+                if (kind === 'supervisor' && (result as SupervisorStatusResponse.AsObject).ok) {
+                    return;
+                }
+                if (kind === 'content' && (result as ContentStatusResponse.AsObject).available) {
```

Comment:
maybe we should rename `available` to `ok` as well. Or rename the `ok` fields to `available`. Either way, it should be consistent :)